kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/importer/sink/PipelineDataSourceSink.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.shardingsphere.data.pipeline.core.exception.job.PipelineImporterJobWriteException;
 import org.apache.shardingsphere.data.pipeline.core.importer.DataRecordMerger;
 import org.apache.shardingsphere.data.pipeline.spi.ratelimit.JobRateLimitAlgorithm;
+import org.apache.shardingsphere.sharding.api.config.cache.ShardingCacheOptionsConfiguration;
```

Comment:
Looks this import of `ShardingCacheOptionsConfiguration` is not used here, it could be removed. You could try CheckStyle plugin and re-import, and then Maven install modified module.

And also in `PipelineImportSQLBuilderTest`.

Reply:
i get it ! thank you help me to participate project！